packages/sdk/README.md

```diff
@@ -38,13 +52,13 @@ All requests to the Looker API server require an access token. For browser imple
 
 Authenticating for the browser takes more setup than authenticating for use with a Node application.
 
-The stand-alone version of the [Looker API Explorer](/packages/api-explorer) uses OAuth and the `BrowserSDK` to get an authentication token for Looker API requests.
+The stand-alone version of the [Looker API Explorer](../../packages/api-explorer) uses OAuth and the `BrowserSDK` to get an authentication token for Looker API requests.
```

Comment:
../../packages/api-explorer can be ../api-explorer?

Reply:
I should have had you do these updates 👍🏼